### PR TITLE
Update templates for composite binding data and nested data samples

### PR DESCRIPTION
**Pull request by mddragnev**
closing #392

### CODE REVIEW

File: code-gen-library/WebGridCompositeAddressEditCellTemplate/WebComponents.ts [hunk 1]
```diff
@@ -23,23 +23,13 @@ public webGridCompositeAddressEditCellTemplate = (ctx: IgcCellTemplateContext) =
...
-         </div>
-         <br>
-     </div>`;
+    return html`
+                <div class="contact-container--edit" style="padding: 1rem">         
+                    <igc-input id="Country" label='Country' type="text" name="country" value="${ctx.cell.row.data.Country}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                    <igc-input id="City" label='City' type="text" name="city" value="${ctx.cell.row.data.City}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                    <igc-input id="PostalCode" label='PostalCode' type="text" name="postalcode" value="${ctx.cell.row.data.PostalCode}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                    <igc-input id="Phone" label='Phone' type="text" name="phone" value="${ctx.cell.row.data.Phone}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                </div>`;
```
Comment (HUSSAR-mtrela) [inline code-gen-library/WebGridCompositeAddressEditCellTemplate/WebComponents.ts:32]: shouldn't there be import for IgcInputComponent like you do in React?

File: code-gen-library/WebGridCompositeContactEditCellTemplate/WebComponents.ts [hunk 2]
```diff
@@ -22,20 +22,11 @@ public webGridCompositeContactEditCellTemplate = (ctx: IgcCellTemplateContext) =
...
-         <div>
-             <strong>Company:</strong>
-             <input id='CompanyName' @keyup=${(e: any) => keyUpHandler(e, ctx)} value='${cell.row.data.CompanyName}'></input>
-         </div>
-     </div>`;
+    return html`<div class="contact-container--edit" style="padding: 1rem">         
+                    <igc-input id="ContactName" label='Name' type="text" name="name" value="${ctx.cell.row.data.ContactName}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                    <igc-input id="ContactTitle" label='Title' type="text" name="title" value="${ctx.cell.row.data.ContactTitle}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                    <igc-input id="CompanyName" label='Company' type="text" name="company" value="${ctx.cell.row.data.CompanyName}" @keyup=${(e: any) => keyUpHandler(e, ctx)}></igc-input>
+                </div>`;
```
Comment (HUSSAR-mtrela) [inline code-gen-library/WebGridCompositeContactEditCellTemplate/WebComponents.ts:29]: shouldn't there be import for IgcInputComponent like you do in React?

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: code-gen-library/WebGridCompositeAddressEditCellTemplate/WebComponents.ts (hunk 1 above)
Comment (mddragnev) [reply code-gen-library/WebGridCompositeAddressEditCellTemplate/WebComponents.ts:32]: No, they dont need importing neither in WC nor in Blazor

File: code-gen-library/WebGridCompositeContactEditCellTemplate/WebComponents.ts (hunk 2 above)
Comment (mddragnev) [reply code-gen-library/WebGridCompositeContactEditCellTemplate/WebComponents.ts:29]: No, they dont need importing neither in WC nor in Blazor

**Review by HUSSAR-mtrela — APPROVED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**